typings/vscode-mssql.d.ts

```diff
@@ -499,10 +499,10 @@ declare module 'vscode-mssql' {
 	}
 
 	/**
-	* Interface containing deployment options of integer type, value property holds values from <DacFx>\Product\Source\DeploymentApi\ObjectTypes.cs enum
+	* Interface containing deployment options of string[] type, value property holds enum names from <DacFx>\Product\Source\DeploymentApi\ObjectTypes.cs enum
```

Comment:
Is it really the enum names? Your description implies it's the display name instead.

Reply:
It holds the enum names, let me update the description clearly :p